File: 250907/숫자들의 배수/multiple-of-numbers.py
n=int(input())
arr=[]
num=0
a=1
while num!=2:
   ele=n*a
   arr.append(ele)
   a+=1
   if ele%5==0:
    num+=1
for i in range(len(arr)):
    print(arr[i],end=" ")

File: 260416/숫자들의 배수/multiple-of-numbers.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin>>n;
    int num=0;
    int arr[100];
    for(int i=0;i<100;i++){
        arr[i]=n*(i+1);
        cout<<arr[i]<<" ";
        if (arr[i]%5==0){
            num+=1;
        }
        if (num==2){
            break;
        }
    }
    // Please write your code here.
    return 0;
}